app/scripts/controllers/eventDetails.js

```diff
@@ -655,11 +665,27 @@ angular
         return currentMinistry ? currentMinistry.activities : [];
       };
 
+      $scope.getEventTypes = () => {
+        const currentMinistry =
+          $scope.ministries &&
+          $scope.ministries.find((m) => m.id === $scope.conference.ministry);
+        const currentPurpose =
+          $scope.ministryPurposes &&
+          $scope.ministryPurposes.find((m) => m.id === $scope.conference.type);
+        return currentMinistry.eventTypes &&
+          currentPurpose &&
+          ((currentPurpose.name && currentPurpose.name.includes('Mission')) ||
+            currentPurpose.name.includes('Conference'))
+          ? currentMinistry.eventTypes
+          : [];
+      };
+
       $scope.$watch(
         'conference.ministry',
         function (newVal, oldVal) {
           if (newVal !== oldVal) {
             $scope.conference.strategy = null;
+            $scope.conference.eventType = null;
```

Comment:
You probably want to add a `$watch` to the ministry Purpose field, in case the admin changes the value, we would want the event type field to disappear.

Reply:
Ah good catch. I'll add that.